static/js/debugging.ts

```diff
@@ -79,6 +79,7 @@ export function load_variables(variables: any) {
       if (variables[i][1]) {
         const variableName = variables[i][0].replace(/^_/, '');
         const role = programData?.variables[variableName];
+        const showRoles = programData?.show_roles;
```

Comment:
I think you're having a problem with this, because you need to add `program_data`to the list of variables!

Reply:
I already had `const role = programData?.variables[variableName] ` in the code when I still used the feature flag in the frontend, then things worked fine...